clicache/test2/CMakeLists.txt

```diff
@@ -63,3 +61,12 @@ if(NOT "${STRONG_NAME_KEY}" STREQUAL "")
 endif()
 
 add_dependencies(Apache.Geode.Tests2 nuget-restore)
+
+enable_testing()
+
+add_test(NAME Apache.Geode.Tests2
```

Comment:
Nice to see more tests running through ctest!

Reply:
Its nice to have consistency so you don't need to know all the different magic incantations for each of the different .NET test frameworks.